nomination/templates/nomination/url_add.html

```diff
@@ -238,6 +249,13 @@ <h4 class="panel-title">Add a Site/URL</h4>
                 initForm();
                 focusAddForm();
                 bindPreviewURL();
+                {% if metadata_vals %}
+                    {% for met,vals in metadata_vals %}
+                        {% if met.form_type == "select" or met.form_type == "selectsingle" %}
+                            new SlimSelect({select: '#{{ met.metadata.name }}'});
+                        {% endif %}
+                    {% endfor %}
+                {% endif %}
```

Comment:
There is a `form_types` variable passed to the template from the view and in line 230 it is made into a JavaScript variable. We could possibly use straight JavaScript then (or whatever best practice JavaScript looks like :D):
```
                for (const [key, value] of Object.entries(form_types)) {
                    if (value == 'select' || value == 'selectsingle'){
                        new SlimSelect({select: document.getElementById(key)});
                    }
```
Or if for some reason this is not desirable, using the `form_types` template variable may still be prettier than `metadata_vals` variable.

Reply:
huh, that variable isn't even in use anymore. Probably something that should have been deleted but never was, lucky for us!

Reply:
ah, probably not used cause it's not that useful as an object and not a dictionary :smile:

Reply:
It's used in nomination/static/nomination/scripts/local.js.

Reply:
or maybe my javascript just needs brushing up...